### PR TITLE
add "join table" for visits

### PR DESCRIPTION
**Pull request by mohamedelshamy55**
In this project, you will add some many-to-many relationships and write more complex queries to use to answer questions.
Understand the different types of relationships between tables.
Prepare complex queries that answer analytical questions.
Use primary key & foreign key mechanisms for joining tables.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -73,3 +73,32 @@ SELECT name, escape_attempts, full_name FROM animals JOIN owners ON animals.owne
...
+SELECT animals.name, visits.date_of_visit FROM animals JOIN visits ON animals.id = visits.animals_id JOIN vets ON vets.id = visits.vets_id WHERE visits.date_of_visit BETWEEN '2020-04-01' AND '2020-08-30' AND vets.name = 'Stephanie Mendez';
+
+/* What animal has the most visits to vets? */
+SELECT animals.name, COUNT(animals.id) FROM animals JOIN visits ON animals.id = visits.animals_id JOIN vets ON vets.id = visits.vets_id GROUP BY animals.name ORDER BY COUNT(animals_id) DESC LIMIT 1;
+
+/* Who was Maisy Smith's first visit? */
+SELECT animals.name, visits.date_of_visit FROM animals JOIN visits ON animals.id = visits.animals_id JOIN vets ON vets.id = visits.vets_id WHERE vets.name = 'Maisy Smith' ORDER BY visits.date_of_visit ASC LIMIT 1;
+
+/* Details for most recent visit: animal information, vet information, and date of visit. */
+SELECT animals.name AS animal_name, animals.date_of_birth, animals.escape_attempts, animals.neutered, animals.weight_kg, vets.name AS vet_name, vets.age, vets.date_of_graduation, visits.date_of_visit FROM animals JOIN visits ON animals.id = visits.animals_id JOIN vets ON vets.id = visits.vets_id ORDER BY visits.date_of_visit DESC LIMIT 1;
```
Comment (ZeenatLawal) [inline queries.sql]: - [OPTIONAL] To get the Details for most recent visit: animal information, vet information, and date of visit without having to list the name of each column, it'll be good if you write like this:
```sql
 SELECT animals.*, vets.*, vistis.visit_date
  FROM visits
  LEFT JOIN animals ON animals.id = visits.animal_id
  LEFT JOIN vets ON vets.id = visits.vet_id
  ORDER BY visits.visit_date DESC
  LIMIT 1;
```

**Review by ZeenatLawal — APPROVED** [review]
Hi @mohamedelshamy55, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 
6vEo


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: queries.sql (hunk 1 above)
Comment (mohamedelshamy55) [reply queries.sql]: thank you for your notes

**Comment by mohamedelshamy55** [comment]
@ZeenatLawal Thank you for finding time to review my project. I appreciate.

**Pull request merged**